### PR TITLE
Import updated Folder / File icons assets and remove the old ones.

### PR DESCRIPTION
**Pull request by crousselle**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @crousselle has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 1]
```diff
@@ -24,31 +24,33 @@ + (UIImage *)box_iconForItem:(BOXItem *)item
         if (item.isFolder) {
             BOXFolder *folder = (BOXFolder *)item;
             if (folder.isExternallyOwned == BOXAPIBooleanYES) {
-                icon = [UIImage imageNamed:@"icon-folder-external"
+                icon = [UIImage imageNamed:@"External"
```
Comment (thuyBox) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: is there an underlying issue that this commit fixes?

Comment (crousselle) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: not sure to understand the question. 
This commit brings the new filetype icons that desin gave us.

Comment (thuyBox) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: I see, we used to have one icon per file type, which will have many duplicates, so now we have one icon per group of file types (group will be like Audio, Video), no need for more granular types of icons.

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 2]
```diff
@@ -24,31 +24,33 @@ + (UIImage *)box_iconForItem:(BOXItem *)item
...
                                   inBundle:browseResourceBundle
              compatibleWithTraitCollection:nil];
             } else if (folder.hasCollaborations == BOXAPIBooleanYES) {
-                icon = [UIImage imageNamed:@"icon-folder-shared"
+                icon = [UIImage imageNamed:@"shared_folder"
```
Comment (thuyBox) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: dont see shared_folder icon in this commit

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m (hunk 1 above)
Comment (crousselle) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: yes, exactly.

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m (hunk 2 above)
Comment (crousselle) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: double checked, it's there, somehow github is overwhelmed by the number of file changed

**Comment by thuyBox** [comment]
+2

**Comment by scottliubox** [comment]
+2

**Pull request merged**